RecoMuon/StandAloneTrackFinder/src/StandAloneTrajectoryBuilder.cc

```diff
@@ -278,11 +278,7 @@ StandAloneMuonTrajectoryBuilder::trajectories(const TrajectorySeed& seed){
   
   TrajectorySeed seedForBW;
 
-  if(theBWSeedType == "noSeed") {
-    TrajectorySeed seedVoid;
-    seedForBW = seedVoid;
-  }
-  else if(theBWSeedType == "fromFWFit") {
+  if(theBWSeedType == "fromFWFit") {
```

Comment:
this looks incorrect: "noSeed" will now go to the final "else" with a LogWarning.
The expected behavior should the that ``` theBWSeedType == "noSeed" ``` is a no-op